### PR TITLE
Fix royal jelly

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
```SUMMARY: Bugfixes "Fix royal jelly"```

#### Purpose of change
Royal jelly is mostly fancy honey and doesn't have magic health properties

#### Describe the solution
- Transfer cureall property from royale jelly to panacea
- Replace royal jelly in labs by panacea
- Remove royal jelly from purifier recipe
- Remove royal jelly from meth recipe (really??)
- Fix panacea to use json use_action instead of a hardcoded one

#### Testing
Get poisoned
Take panacea
Be cured

#### Additional context
Maybe we could expend the list of effect cureall cures?

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
Panacea is a fictional drug made for a Bright Nights mod. It shouldn't spawn in vanilla.

**Comment by Amoebka** [comment]
Royal Jelly item in the game explicitly refers to jelly produced by **giant blob-mutated bees**, hence the "magical effects". It's supposed to be the major reward for looting giant hives.

File: data/json/items/comestibles/other.json [hunk 1]
```diff
@@ -70,17 +70,15 @@
...
     "weight": "150 g",
     "color": "white",
-    "use_action": "ROYAL_JELLY",
     "comestible_type": "FOOD",
     "symbol": "%",
-    "healthy": 10,
+    "healthy": 1,
     "calories": 240,
-    "description": "A translucent hexagonal chunk of wax, filled with dense, milky jelly.  Delicious, and rich with the most beneficial substances the hive can produce, it is useful for curing all sorts of afflictions.",
+    "description": "A translucent hexagonal chunk of wax, filled with dense, milky jelly.  Delicious, and rich with the most beneficial substances the hive can produce, contrary to some belief it doesn't cure anything.",
```
Comment (tenmillimaster) [inline data/json/items/comestibles/other.json:80]:
```suggestion
    "description": "A translucent hexagonal chunk of wax, filled with dense, milky jelly.  Contrary to some belief it doesn't cure anything.   Still, it is delicious and rich with the most beneficial substances the hive can produce.",
```

Last sentence seemed awkward, though I dunno if this is much of an improvement.

Comment (anothersimulacrum) [reply data/json/items/comestibles/other.json:80]:
```suggestion
    "description": "A translucent hexagonal chunk of wax, filled with dense, milky jelly.  Though some hold it as a panacea, it doesn't have any medical benefit.  Still, it is delicious, and rich with the most beneficial substances the hive can produce..",
```
Still a bit awkward and clashes a tiny bit with the beneficial substances bit.

**Comment by wapcaplet** [comment]
@Fris0uman There is a test case for the cure-all effect in `iuse_test.cpp` that should be straightforward to replace with `panacea` as well.

**Comment by Night-Pryanik** [comment]
@kevingranade why this was merged? What about my objection?

**Comment by kevingranade** [comment]
I agree with the royal jelly change, I'm fine with it having *some* extreme effect, but being a cureall is a really weird one. This has strictly stuck around up til now because it was just some random legacy thing.
The spawn of Royal Jelly in the chem lab item group didn't make sense either. Frankly swapping panacea in there is still a bit odd, but it's an improvement. Ideally I'd like to see it be promoted to be a medical lab finale or a quest result instead of just appearing at random mixed in with ordinary lab supplies.

I see this as two changes:
A magic cureall no longer appears in otherwise undeveloped beehive locations.
The magic cureall that appears sometimes in labs has been palette swapped to be a sci-fi flavored magic curall instead of an alien mutation flavored magic cureall.

Whether panacea was added for AfterShock or not is kind of irrelevant, mechanically it's defined in the base game, and as far as *existing*, it doesn't conflict with lore, so it's fine.  I'd also be ok with not having a panacea medicine in the vanilla game at all, but I don't feel like pushing either for or against.
If someone wanted to broaden available medicines and treatment options, I'd flip to wanting to remove it, but with the options we have now removing it is pretty meh.

**Comment by MirrorMorphMemories** [comment]
As far as what possible extreme effect Royal Jelly could have, maybe something that ties into the mutation system? I think the old justification for the CureAll effect was that Royal Jelly's existence had something to do with extradimensional blob biology magic, and the accelerated healing rates survivors used to have was the result of said blob biology magic, so it was plausible that they could interact in an interesting way. Now that that's gone, the game could instead tie Royal Jelly to the other blob biology magic in the game, mutations. 

I'm not sure yet what would be the best interaction possible - rerolling mutations? A weaker form of the Purifier Smart Shot? Simply being an early game mutagen/targeted mutagen option? - but it would be thematic, mechanically interesting, and makes sense if you're willing to accept the blob violating normal laws of biology. Maybe we'd even be able to tie in Apis to this - where does she come from? Does she have something to do with Royal Jelly? Do both of them have something to do with the experiments that caused the Cataclysm, instead of coming afterwards? Is this all somehow related to the mystery behind the real world Colony Collapse Disorder?

Failing that... Royal Jelly could have something to do with CBMs? Maybe it's the immunosuppressant  your body needs to avoid rejecting your CBMs if you exceed the CBM slot limit... there's lots of room for biology magic to be useful. It just takes biology knowledge, which I'm desperately trying to remember from high school Bio class.

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/tips-tricks-and-newb-questions-reborn/16870/2055

**Pull request merged**